README.md

```diff
@@ -13,37 +15,52 @@ npm install restful.js
 
 Include `restful.min.js` to the HTML, and the `restful` object is now available in the global scope:
 
+Standalone version:
 ```html
 <script type="text/javascript" src="/path/to/bower_components/restful.js/dist/restful.min.js"></script>
 ```
 
-Alternately, you can use [RequireJS](http://requirejs.org/) or [Browserify](http://browserify.org/) to avoid global scoping.
+Pre built with fetch version:
+```html
+<script type="text/javascript" src="/path/to/bower_components/restful.js/dist/restful.fetch.min.js"></script>
+```
+
+Alternately, you can use a module loader like [webpack](http://webpack.github.io/).
 
 ```js
-var restful = require('restful.js');
+import restful from 'restful.js';
 ```
 
 ## Usage
 
 ### Create a resource targeting your API
 
-Start by defining the base endpoint for an API, for instance `http://api.example.com`.
+Restful.js needs an http backend in order to perform queries. Two http backend are currently available:
+* [fetch](https://github.com/github/fetch): For using restful.js in a browser.
+* [request](https://github.com/request/request): For using restful.js in a browser.
 
+There are defined as optional dependencies and therefore you must install them with npm if you need them.
```

Comment:
... you must install them either with `npm` or `bower` depending your package manager. Standalone version already embeds `request`.